diff --git a/components/footer.tsx b/components/footer.tsx
index 5a03526..f5ec11b 100644
--- a/components/footer.tsx
+++ b/components/footer.tsx
@@ -1,25 +1,39 @@
 import Container from "./container";
-import { CONTACT_EMAIL } from "../lib/constants";
+import { CONTACT_EMAIL, MENUS } from "../lib/constants";
+import Link from "next/link";
 const Footer = () => {
   return (
     <footer className="bg-neutral-50 border-t border-neutral-200">
       <Container>
-        <div className="py-28 flex flex-col lg:flex-row items-center">
-          <h3 className="text-4xl lg:text-[2.5rem] font-bold tracking-tighter leading-tight text-center lg:text-left mb-10 lg:mb-0 lg:pr-4 lg:w-1/2 cc__textColor">
-            Share and Subscribe
-          </h3>
-          <div className="flex flex-col lg:flex-row justify-center items-center lg:pl-4 lg:w-1/2">
-            <a
-              href={`mailto:${CONTACT_EMAIL}`}
-              className="mx-3 bg-black hover:bg-white hover:text-black border-2 cc__borderColor text-white font-bold py-3 px-12 lg:px-8 duration-200 transition-colors mb-6 lg:mb-0 cc__bgColor"
-            >
-              Contact Us
-            </a>
+        {/* <div className="py-20 flex flex-col lg:flex-row items-center justify-around"> */}
+        <div className="py-20 flex flex-col md:flex-row justify-around">
+          <div className="flex flex-col font-bold tracking-tighter leading-tight mb-4 lg:mb-0">
+            {/* <h3 className="text-xl text-center lg:text-left mb-4 lg:pr-4 cc__textColor"> */}
+            <h3 className="text-xl mb-4 lg:pr-4 cc__textColor">Read More</h3>
+            {MENUS.map((menu) => {
+              return (
+                <Link href={menu.path} key={menu.id}>
+                  {menu.menuName}
+                </Link>
+              );
+            })}
+          </div>
+          <div className="flex flex-col font-bold tracking-tighter leading-tight mb-4 lg:mb-0">
+            {/* <h3 className="text-xl text-center lg:text-left mb-4 lg:pr-4 cc__textColor"> */}
+            <h3 className="text-xl mb-4 lg:pr-4 cc__textColor">
+              Important Links
+            </h3>
+            <Link href="/about">About Us</Link>
+            <Link href="/privacy-policy">Privacy Policy</Link>
+            <Link href={"/contact"}>Contact Us</Link>
+          </div>
+          {/* <div className="flex flex-col lg:flex-row justify-center items-center lg:pl-4 lg:w-1/2"> */}
+          <div className="flex items-center lg:pl-4">
             <a
               href={`mailto:${CONTACT_EMAIL}`}
-              className="mx-3 font-bold hover:underline"
+              className="lg:mx-3 bg-black hover:bg-white hover:text-black border-2 cc__borderColor text-white font-bold py-3 px-12 lg:px-8 duration-200 transition-colors cc__bgColor"
             >
-              Send us a email
+              Send us an email
             </a>
           </div>
         </div>
diff --git a/components/meta.tsx b/components/meta.tsx
index 49be8b7..d351d91 100644
--- a/components/meta.tsx
+++ b/components/meta.tsx
@@ -12,12 +12,12 @@ const Meta = () => {
       <link rel="shortcut icon" href="/favicon/favicon.ico" />
       <meta name="msapplication-TileColor" content="#000000" />
       <meta name="msapplication-config" content="/favicon/browserconfig.xml" />
-      <meta name="theme-color" content="#000" />
+      <meta name="theme-color" content="#00aefd" />
       <meta
         name="Basic Knowledge"
         content={`A Blog site for all your basic knowledge needs.`}
       />
-      <meta property="og:image" content={HOME_OG_IMAGE_URL} />
+      {/* <meta property="og:image" content={HOME_OG_IMAGE_URL} /> */}
     </Head>
   );
 };
diff --git a/components/more-stories.tsx b/components/more-stories.tsx
index 135af2a..c781a7d 100644
--- a/components/more-stories.tsx
+++ b/components/more-stories.tsx
@@ -5,13 +5,15 @@ import Button from "./button";
 
 type Props = {
   posts: Post[];
+  dir: string;
 };
 
-const MoreStories = ({ posts }: Props) => {
+const MoreStories = ({ posts, dir }: Props) => {
   return (
     <section>
       <h2 className="mb-8 text-5xl md:text-7xl font-bold tracking-tighter leading-tight cc__textColor">
-        Top Stories
+        Top {dir == "posts" ? "general" : dir == "home" ? "trending" : dir}{" "}
+        stories
       </h2>
       <div className="grid grid-cols-1 md:grid-cols-2 md:gap-x-16 lg:gap-x-32 gap-y-20 md:gap-y-8 mb-16">
         {posts.map((post) => (
@@ -23,10 +25,10 @@ const MoreStories = ({ posts }: Props) => {
             author={post.author}
             slug={post.slug}
             excerpt={post.excerpt}
-            dir={"home"}
+            dir={dir}
           />
         ))}
-        <Button text="More Storiess" link="/posts" />
+        {/* <Button text="More Storiess" link={`/${dir}`} /> */}
       </div>
     </section>
   );
diff --git a/components/navbar.tsx b/components/navbar.tsx
index 369c772..7771217 100644
--- a/components/navbar.tsx
+++ b/components/navbar.tsx
@@ -2,18 +2,11 @@ import React from "react";
 import Link from "next/link";
 import { useRouter } from "next/router";
 import Image from "next/image";
+import { MENUS } from "../lib/constants";
 
 const Navbar = () => {
   const [navbarOpen, setNavbarOpen] = React.useState(false);
   const router = useRouter();
-  const MENUS = [
-    { id: 0, menuName: "Home", path: "/" },
-    { id: 1, menuName: "All Posts", path: "/posts" },
-    { id: 2, menuName: "Education", path: "/education" },
-    { id: 3, menuName: "Politics", path: "/politics" },
-    { id: 4, menuName: "Sports", path: "/sports" },
-    { id: 5, menuName: "Gadgets", path: "/gadgets" },
-  ];
   return (
     <>
       <nav className="relative flex flex-wrap items-center justify-between px-2 py-3 bg-gray-100 mb-3">
diff --git a/lib/constants.ts b/lib/constants.ts
index b7e8a57..6e176c2 100644
--- a/lib/constants.ts
+++ b/lib/constants.ts
@@ -2,4 +2,13 @@ export const EXAMPLE_PATH = "basic-Knowledge";
 export const HOME_OG_IMAGE_URL =
   "https://images.unsplash.com/photo-1661956600684-97d3a4320e45?ixlib=rb-4.0.3&ixid=MnwxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80";
 export const YOUTUBE_LINK = "https://www.youtube.com";
+export const MENUS = [
+  { id: 0, menuName: "Home", path: "/" },
+  { id: 1, menuName: "All Posts", path: "/posts" },
+  { id: 2, menuName: "Education", path: "/education" },
+  { id: 3, menuName: "Politics", path: "/politics" },
+  { id: 4, menuName: "Sports", path: "/sports" },
+  { id: 5, menuName: "Gadgets", path: "/gadgets" },
+];
 export const CONTACT_EMAIL = "basicknowledge544@gmail.com";
+export const WEBSITE_URL = "https://www.basicknowledge.co.in";
diff --git a/pages/education/[slug].tsx b/pages/education/[slug].tsx
index bac6627..f021510 100644
--- a/pages/education/[slug].tsx
+++ b/pages/education/[slug].tsx
@@ -10,9 +10,10 @@ import PostTitle from "../../components/post-title";
 import Head from "next/head";
 import markdownToHtml from "../../lib/markdownToHtml";
 import type PostType from "../../interfaces/post";
+import { WEBSITE_URL } from "../../lib/constants";
 
 const directory = "_education";
-
+console.log(__dirname);
 type Props = {
   post: PostType;
   morePosts: PostType[];
@@ -35,7 +36,12 @@ export default function Post({ post, morePosts, preview }: Props) {
             <article className="mb-32">
               <Head>
                 <title>{post.title}</title>
-                <meta property="og:image" content={post.ogImage.url} />
+                <meta property="og:title" content={post.title} />
+                <meta property="og:description" content={post.excerpt} />
+                <meta
+                  property="og:image"
+                  content={`${WEBSITE_URL}${post.ogImage.url}`}
+                />
               </Head>
               <PostHeader
                 title={post.title}
@@ -61,7 +67,16 @@ type Params = {
 export async function getStaticProps({ params }: Params) {
   const post = getPostBySlug(
     params.slug,
-    ["title", "date", "slug", "author", "content", "ogImage", "coverImage"],
+    [
+      "title",
+      "date",
+      "slug",
+      "author",
+      "content",
+      "ogImage",
+      "coverImage",
+      "excerpt",
+    ],
     directory
   );
   const content = await markdownToHtml(post.content || "");
diff --git a/pages/gadgets/[slug].tsx b/pages/gadgets/[slug].tsx
index 83a6a19..37a6231 100644
--- a/pages/gadgets/[slug].tsx
+++ b/pages/gadgets/[slug].tsx
@@ -10,6 +10,7 @@ import PostTitle from "../../components/post-title";
 import Head from "next/head";
 import markdownToHtml from "../../lib/markdownToHtml";
 import type PostType from "../../interfaces/post";
+import { WEBSITE_URL } from "../../lib/constants";
 
 const directory = "_gadgets";
 
@@ -35,7 +36,12 @@ export default function Post({ post, morePosts, preview }: Props) {
             <article className="mb-32">
               <Head>
                 <title>{post.title}</title>
-                <meta property="og:image" content={post.ogImage.url} />
+                <meta property="og:title" content={post.title} />
+                <meta property="og:description" content={post.excerpt} />
+                <meta
+                  property="og:image"
+                  content={`${WEBSITE_URL}${post.ogImage.url}`}
+                />
               </Head>
               <PostHeader
                 title={post.title}
@@ -61,7 +67,16 @@ type Params = {
 export async function getStaticProps({ params }: Params) {
   const post = getPostBySlug(
     params.slug,
-    ["title", "date", "slug", "author", "content", "ogImage", "coverImage"],
+    [
+      "title",
+      "date",
+      "slug",
+      "author",
+      "content",
+      "ogImage",
+      "coverImage",
+      "excerpt",
+    ],
     directory
   );
   const content = await markdownToHtml(post.content || "");
diff --git a/pages/home/[slug].tsx b/pages/home/[slug].tsx
index 0e2cd71..a4d88c2 100644
--- a/pages/home/[slug].tsx
+++ b/pages/home/[slug].tsx
@@ -10,6 +10,7 @@ import PostTitle from "../../components/post-title";
 import Head from "next/head";
 import markdownToHtml from "../../lib/markdownToHtml";
 import type PostType from "../../interfaces/post";
+import { WEBSITE_URL } from "../../lib/constants";
 
 const directory = "_home";
 
@@ -35,7 +36,12 @@ export default function Post({ post, morePosts, preview }: Props) {
             <article className="mb-32">
               <Head>
                 <title>{post.title}</title>
-                <meta property="og:image" content={post.ogImage.url} />
+                <meta property="og:title" content={post.title} />
+                <meta property="og:description" content={post.excerpt} />
+                <meta
+                  property="og:image"
+                  content={`${WEBSITE_URL}${post.ogImage.url}`}
+                />
               </Head>
               <PostHeader
                 title={post.title}
@@ -61,7 +67,16 @@ type Params = {
 export async function getStaticProps({ params }: Params) {
   const post = getPostBySlug(
     params.slug,
-    ["title", "date", "slug", "author", "content", "ogImage", "coverImage"],
+    [
+      "title",
+      "date",
+      "slug",
+      "author",
+      "content",
+      "ogImage",
+      "coverImage",
+      "excerpt",
+    ],
     directory
   );
   const content = await markdownToHtml(post.content || "");
diff --git a/pages/index.tsx b/pages/index.tsx
index 566ede1..1f7df7f 100644
--- a/pages/index.tsx
+++ b/pages/index.tsx
@@ -11,9 +11,22 @@ import MoreStories from "../components/more-stories";
 type Props = {
   allPosts: Post[];
   featuredPosts?: Post[];
+  featuredEducationPosts?: Post[];
+  featuredSportsPosts?: Post[];
+  featuredPoliticsPosts?: Post[];
+  featuredGadgetsPosts?: Post[];
+  featuredGeneralPosts?: Post[];
 };
 
-export default function Index({ allPosts, featuredPosts }: Props) {
+export default function Index({
+  allPosts,
+  featuredPosts,
+  featuredEducationPosts,
+  featuredSportsPosts,
+  featuredPoliticsPosts,
+  featuredGadgetsPosts,
+  featuredGeneralPosts,
+}: Props) {
   const heroPost = featuredPosts[0] || allPosts[0];
   const morePosts = allPosts.slice(1, 5);
   return (
@@ -34,7 +47,22 @@ export default function Index({ allPosts, featuredPosts }: Props) {
               excerpt={heroPost.excerpt}
             />
           )}
-          {morePosts.length > 0 && <MoreStories posts={morePosts} />}
+          {morePosts.length > 0 && <MoreStories posts={morePosts} dir="home" />}
+          {featuredEducationPosts.length > 0 && (
+            <MoreStories posts={featuredEducationPosts} dir="education" />
+          )}
+          {featuredSportsPosts.length > 0 && (
+            <MoreStories posts={featuredSportsPosts} dir="sports" />
+          )}
+          {featuredPoliticsPosts.length > 0 && (
+            <MoreStories posts={featuredPoliticsPosts} dir="politics" />
+          )}
+          {featuredGadgetsPosts.length > 0 && (
+            <MoreStories posts={featuredGadgetsPosts} dir="gadgets" />
+          )}
+          {featuredGeneralPosts.length > 0 && (
+            <MoreStories posts={featuredGeneralPosts} dir="posts" />
+          )}
         </Container>
       </Layout>
     </>
@@ -42,14 +70,46 @@ export default function Index({ allPosts, featuredPosts }: Props) {
 }
 
 export const getStaticProps = async () => {
-  const directory = "_home";
   const allPosts = getAllPosts(
     ["title", "date", "slug", "author", "coverImage", "excerpt", "featured"],
-    directory
+    "_home"
+  );
+  const allEducationPosts = getAllPosts(
+    ["title", "date", "slug", "author", "coverImage", "excerpt", "featured"],
+    "_education"
+  );
+  const allPoliticsPosts = getAllPosts(
+    ["title", "date", "slug", "author", "coverImage", "excerpt", "featured"],
+    "_politics"
+  );
+  const allSportsPosts = getAllPosts(
+    ["title", "date", "slug", "author", "coverImage", "excerpt", "featured"],
+    "_sports"
+  );
+  const allGadgetsPosts = getAllPosts(
+    ["title", "date", "slug", "author", "coverImage", "excerpt", "featured"],
+    "_gadgets"
+  );
+  const allGeneralPosts = getAllPosts(
+    ["title", "date", "slug", "author", "coverImage", "excerpt", "featured"],
+    "_posts"
   );
   const featuredPosts = getFeaturedPosts(allPosts);
+  const featuredEducationPosts = getFeaturedPosts(allEducationPosts);
+  const featuredPoliticsPosts = getFeaturedPosts(allPoliticsPosts);
+  const featuredSportsPosts = getFeaturedPosts(allSportsPosts);
+  const featuredGadgetsPosts = getFeaturedPosts(allGadgetsPosts);
+  const featuredGeneralPosts = getFeaturedPosts(allGeneralPosts);
 
   return {
-    props: { allPosts, featuredPosts },
+    props: {
+      allPosts,
+      featuredPosts,
+      featuredEducationPosts,
+      featuredSportsPosts,
+      featuredPoliticsPosts,
+      featuredGadgetsPosts,
+      featuredGeneralPosts,
+    },
   };
 };
diff --git a/pages/politics/[slug].tsx b/pages/politics/[slug].tsx
index 9b31c05..0bbce7b 100644
--- a/pages/politics/[slug].tsx
+++ b/pages/politics/[slug].tsx
@@ -10,6 +10,7 @@ import PostTitle from "../../components/post-title";
 import Head from "next/head";
 import markdownToHtml from "../../lib/markdownToHtml";
 import type PostType from "../../interfaces/post";
+import { WEBSITE_URL } from "../../lib/constants";
 
 const directory = "_politics";
 
@@ -35,7 +36,12 @@ export default function Post({ post, morePosts, preview }: Props) {
             <article className="mb-32">
               <Head>
                 <title>{post.title}</title>
-                <meta property="og:image" content={post.ogImage.url} />
+                <meta property="og:title" content={post.title} />
+                <meta property="og:description" content={post.excerpt} />
+                <meta
+                  property="og:image"
+                  content={`${WEBSITE_URL}${post.ogImage.url}`}
+                />
               </Head>
               <PostHeader
                 title={post.title}
@@ -61,7 +67,16 @@ type Params = {
 export async function getStaticProps({ params }: Params) {
   const post = getPostBySlug(
     params.slug,
-    ["title", "date", "slug", "author", "content", "ogImage", "coverImage"],
+    [
+      "title",
+      "date",
+      "slug",
+      "author",
+      "content",
+      "ogImage",
+      "coverImage",
+      "excerpt",
+    ],
     directory
   );
   const content = await markdownToHtml(post.content || "");
diff --git a/pages/posts/[slug].tsx b/pages/posts/[slug].tsx
index 5cfdfbf..b756fa8 100644
--- a/pages/posts/[slug].tsx
+++ b/pages/posts/[slug].tsx
@@ -10,6 +10,7 @@ import PostTitle from "../../components/post-title";
 import Head from "next/head";
 import markdownToHtml from "../../lib/markdownToHtml";
 import type PostType from "../../interfaces/post";
+import { WEBSITE_URL } from "../../lib/constants";
 
 type Props = {
   post: PostType;
@@ -33,7 +34,12 @@ export default function Post({ post, morePosts, preview }: Props) {
             <article className="mb-32">
               <Head>
                 <title>{post.title}</title>
-                <meta property="og:image" content={post.ogImage.url} />
+                <meta property="og:title" content={post.title} />
+                <meta property="og:description" content={post.excerpt} />
+                <meta
+                  property="og:image"
+                  content={`${WEBSITE_URL}${post.ogImage.url}`}
+                />
               </Head>
               <PostHeader
                 title={post.title}
@@ -60,7 +66,16 @@ export async function getStaticProps({ params }: Params) {
   const directory = "_posts";
   const post = getPostBySlug(
     params.slug,
-    ["title", "date", "slug", "author", "content", "ogImage", "coverImage"],
+    [
+      "title",
+      "date",
+      "slug",
+      "author",
+      "content",
+      "ogImage",
+      "coverImage",
+      "excerpt",
+    ],
     directory
   );
   const content = await markdownToHtml(post.content || "");
diff --git a/pages/sports/[slug].tsx b/pages/sports/[slug].tsx
index a3096c0..67ee3ea 100644
--- a/pages/sports/[slug].tsx
+++ b/pages/sports/[slug].tsx
@@ -10,6 +10,7 @@ import PostTitle from "../../components/post-title";
 import Head from "next/head";
 import markdownToHtml from "../../lib/markdownToHtml";
 import type PostType from "../../interfaces/post";
+import { WEBSITE_URL } from "../../lib/constants";
 
 type Props = {
   post: PostType;
@@ -33,7 +34,12 @@ export default function Post({ post, morePosts, preview }: Props) {
             <article className="mb-32">
               <Head>
                 <title>{post.title}</title>
-                <meta property="og:image" content={post.ogImage.url} />
+                <meta property="og:title" content={post.title} />
+                <meta property="og:description" content={post.excerpt} />
+                <meta
+                  property="og:image"
+                  content={`${WEBSITE_URL}${post.ogImage.url}`}
+                />
               </Head>
               <PostHeader
                 title={post.title}
@@ -60,7 +66,16 @@ export async function getStaticProps({ params }: Params) {
   const directory = "_sports";
   const post = getPostBySlug(
     params.slug,
-    ["title", "date", "slug", "author", "content", "ogImage", "coverImage"],
+    [
+      "title",
+      "date",
+      "slug",
+      "author",
+      "content",
+      "ogImage",
+      "coverImage",
+      "excerpt",
+    ],
     directory
   );
   const content = await markdownToHtml(post.content || "");
diff --git a/styles/index.css b/styles/index.css
index 3d3f533..b59d657 100644
--- a/styles/index.css
+++ b/styles/index.css
@@ -28,3 +28,35 @@ a {
 /* a:hover {
   text-decoration: underline;
 } */
+
+/* Stylings for Static pages such as Privacy policy, about-us, contact us */
+.staticPage h1 {
+  font-size: 2.5rem;
+  line-height: 2.5rem;
+  margin-top: 2rem;
+  margin-bottom: 1rem;
+}
+.staticPage h2 {
+  font-size: 1.875rem /* 30px */;
+  line-height: 2.25rem /* 36px */;
+  margin-top: 2rem;
+  margin-bottom: 1rem;
+}
+.staticPage h3 {
+  font-size: 1.5rem /* 24px */;
+  line-height: 2rem /* 32px */;
+  margin-top: 2rem;
+  margin-bottom: 1rem;
+}
+
+.staticPage ul {
+  list-style-type: disc;
+  margin-left: 1rem;
+  margin-bottom: 1rem;
+}
+
+.staticPage ol {
+  list-style-type: lower-roman;
+  margin-left: 1rem;
+  margin-bottom: 1rem;
+}
